src/Microsoft.Framework.PackageManager/Restore/RemoteWalkProvider.cs

```diff
@@ -87,6 +87,11 @@ public RemoteWalkProvider(IPackageFeed source, bool isHttp)
                 await nupkgStream.CopyToAsync(stream);
             }
         }
+
+        private static bool IsHttpSource(IPackageFeed source)
+        {
+            return (source is NuGetv2Feed);
```

Comment:
Yeah. I'll change it to `return (source is NuGetv2Feed) || (source is NuGetv3Feed)` when we have that change. Will add TODO comment here.